it/v2/utils/CommonGenerators.scala

```diff
@@ -153,7 +153,7 @@ trait CommonGenerators {
     for {
       status <- Gen.oneOf(MessageStatus.statusValues)
       uri    <- Gen.alphaNumStr
-    } yield MessageUpdate(status, Some(ObjectStoreURI(uri)))
+    } yield MessageUpdate(status, Some(ObjectStoreURI(uri)), Some(MessageType.DeclarationData))
```

Comment:
Can we make the message type random too so that we have a chance of catching issues that might be message type dependent?